### PR TITLE
Release PR for 2.2.2 Patch Release

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/smrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v2.2.1](https://github.com/nf-core/smrnaseq/releases/tag/2.2.1) - 2023-05-08
+## dev
```
Comment (jfy133) [inline CHANGELOG.md:6]: Jumping ahead if this is a patch release?

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/smrnaseq" target="_blank">nf-core/smrnaseq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/smrnaseq/2.2.2dev" target="_blank">nf-core/smrnaseq</a>
```
Comment (jfy133) [inline assets/multiqc_config.yml:2]: Wouldn't this be e.g. 2.2.3?

File: modules/local/parse_fasta_mirna.nf [hunk 3]
```diff
@@ -7,11 +7,11 @@ process PARSE_FASTA_MIRNA {
         'biocontainers/seqkit:2.3.1--h9ee0642_0' }"
 
     input:
-    path fasta
+    tuple val(meta2), path(fasta)
```
Comment (jfy133) [inline modules/local/parse_fasta_mirna.nf:10]: Why meta2 given its the only input channel?

File: CHANGELOG.md (hunk 1 above)
Comment (jfy133) [reply CHANGELOG.md:6]: Missing version bumps for multiQC and fasto for example

Comment (jfy133) [reply CHANGELOG.md:6]: Missing updates to paths for one of the tools (and removing .gz)

**Review by jfy133 — COMMENTED** [review]
Mostly changelog and versions look weird for a release pr

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,16 @@
...
+
+- Remove public_aws_ecr profile
+
+## [v2.2.2](https://github.com/nf-core/smrnaseq/releases/tag/2.2.2) - 2023-06-01
```
Comment (christopher-mohr) [inline CHANGELOG.md:10]: Correct release date?

File: assets/multiqc_config.yml (hunk 2 above)
Comment (christopher-mohr) [reply assets/multiqc_config.yml:2]: Or just `2.2.2` ?

File: nextflow.config [hunk 5]
```diff
@@ -258,8 +271,8 @@ manifest {
     homePage        = 'https://github.com/nf-core/smrnaseq'
     description     = """Small RNA-Seq Best Practice Analysis Pipeline."""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=22.10.1'
-    version         = '2.2.1'
+    nextflowVersion = '!>=23.04.0'
+    version = '2.2.2dev'
```
Comment (christopher-mohr) [inline nextflow.config:275]:
```suggestion
    version = '2.2.2'
```

File: subworkflows/local/genome_quant.nf [hunk 6]
```diff
@@ -3,20 +3,20 @@
...
 
 include { INDEX_GENOME      } from '../../modules/local/bowtie_genome'
-include { BAM_SORT_SAMTOOLS } from '../nf-core/bam_sort_samtools'
+include { BAM_SORT_STATS_SAMTOOLS } from '../nf-core/bam_sort_stats_samtools'
 include { BOWTIE_MAP_SEQ as BOWTIE_MAP_GENOME } from '../../modules/local/bowtie_map_mirna'
 
 workflow GENOME_QUANT {
     take:
-    fasta
+    fasta //channel: [ val(meta2), [ reads ] ]
```
Comment (christopher-mohr) [inline subworkflows/local/genome_quant.nf:11]: I think this is just `fasta` here without meta.

 ```suggestion
    fasta 
```

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
Just some minor things in addition to what James already pointed out.

**Review by grst — APPROVED** [review]
Nothing to add besides what has been said. 
As the last release was 2.2.1 this one should be 2.2.2 or 2.3.0

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,7 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v2.2.1](https://github.com/nf-core/smrnaseq/releases/tag/2.2.1) - 2023-05-08
+## dev
+
+- Remove public_aws_ecr profile
```
Comment (maxulysse) [inline CHANGELOG.md:8]: This should be in the release

File: subworkflows/local/genome_quant.nf [hunk 8]
```diff
@@ -3,20 +3,20 @@
...
 
 include { INDEX_GENOME      } from '../../modules/local/bowtie_genome'
-include { BAM_SORT_SAMTOOLS } from '../nf-core/bam_sort_samtools'
+include { BAM_SORT_STATS_SAMTOOLS } from '../nf-core/bam_sort_stats_samtools'
 include { BOWTIE_MAP_SEQ as BOWTIE_MAP_GENOME } from '../../modules/local/bowtie_map_mirna'
 
 workflow GENOME_QUANT {
     take:
-    fasta
+   fasta
```
Comment (christopher-mohr) [inline subworkflows/local/genome_quant.nf:11]: Not sure if the identation is wrong here?

**Review by christopher-mohr — APPROVED** [review]
One comment, LGTM besides that.

**Pull request merged**